hazelcast/src/main/java/com/hazelcast/map/impl/event/MapEventPublisherImpl.java

```diff
@@ -76,6 +78,9 @@ public MapEventPublisherImpl(MapServiceContext mapServiceContext) {
         } else {
             this.filteringStrategy = new DefaultEntryEventFilteringStrategy(serializationService, mapServiceContext);
         }
+        this.queryCacheEventPublisher = new QueryCacheEventPublisher(filteringStrategy,
```

Comment:
Implementation of methods publishing events to query cache subscribers are gathered in `QueryCacheEventPublisher`.